Car-Specs/Car-Specs/Services/Database/DatabaseContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Car_Specs.Models.Configuration;
+using Car_Specs.Models.Data;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+
+namespace Car_Specs.Services.Database
```

Comment:
The naming convention for namespace is not correct. Car_specs in not a good way to better CarSpecs.

Reply:
Changed